### PR TITLE
Add `python-utils` package

### PR DESCRIPTION
**Pull request by kchenTTP**
Some shortcuts to reduce the amount of boilerplate code for python scripts.

### CODE REVIEW

**Review by smeech — CHANGES_REQUESTED** [review]
Three brief snippets for adding Python boilerplate.
All simple text expansions with no `cmd`, `script`, `shell` or risky code.
Safe to merge after the author, or we, remove the `force_clipboard: true` items.

File: packages/python-utils/0.1.0/package.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+matches:
+  - trigger: ":pymain"
+    force_clipboard: true
```
[the remark this answers is not in the document]
Comment (kchenTTP) [reply packages/python-utils/0.1.0/package.yml]: I've removed `force_clipboard: true` for all the triggers. Let me know if you have any more suggestions.

**Comment by kchenTTP** [comment]
I've also added more triggers on a whim. Please also review those and let me know if there's anything I need to fix. Sorry about that.

**Review by smeech — APPROVED** [review]

**Comment by smeech** [comment]
Excellent! Thank you.

**Pull request merged**